src/_style.scss

```diff
@@ -564,3 +564,20 @@ select.form-control {
   color: $gray-700;
   text-decoration: none;
 }
+
+.dropdown-item:active {
+    background-color: #F2F0EF;
+}
+
+.dropdown-container {
+    box-shadow: 3px 3px 15px #F2F0EF;
```

Comment:
According to design box-shadow should be `box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.15), 0px 2px 8px rgba(0, 0, 0, 0.15)`

Reply:
Also add `border-radius: 4px`